server_tests/test/clientTests.js

```diff
@@ -13,11 +13,22 @@ var Q = require('../node_modules/q');
 var Util = require('../lib/util');
 
 describe("RUST API TEST SUITE", function() {
-    var testClient = new OscClient(process.env.SCARLET_TEST_HOST, process.env.SCARLET_TEST_PORT);
+    var testClient = new OscClient(process.env.SCARLET_TEST_HOST, process.env.SCARLET_TEST_PORT, process.env.SCARLET_TEST_WIFI, process.env.SCARLET_TEST_MODEL);
     var Comparison = new Compare();
     var Utility = new Util(testClient);
     var defaultOptionsFile = './defaults/mock.json';
     var timeoutValue = 30000;
+    var testViaWifi = process.env.SCARLET_TEST_WIFI == 1;
+    var testModel = process.env.SCARLET_TEST_MODEL;
```

Comment:
Should probably be `var testModel = process.env.SCARLET_TEST_MODEL || camModels.MOCKCAM`